core/mondoo-github-security.mql.yaml

```diff
@@ -302,7 +323,7 @@ queries:
   - uid: mondoo-github-repository-security-security-policy
     title: Ensure repository defines a security policy
     impact: 30
-    mql: "if ( github.organization.repositories.one(name == \".github\") ) {\n  github.organization.repositories.where( name == \".github\").all( \n    files.one( name.downcase == \"security.md\")\n  ) || github.repository.files.one( name.downcase == \"security.md\")\n} else {\n  github.repository.files.one( name.downcase == \"security.md\")\n}\n"
+    mql: "github.repository.files.one( name.downcase == \"security.md\")"
```

Comment:
```suggestion
    mql: |
      github.repository.files.one(name.downcase == "security.md")
```

Reply:
can you please adjust also in the section audit line 337 the mql statement to

```coffee
github.repository.files.one( name.downcase == "security.md")
```

Reply:
Both fixed. Thanks for the suggestions.